EarlGrey/Additions/UIViewController+GREYAdditions.m

```diff
@@ -138,14 +138,18 @@ - (void)greyswizzled_viewWillAppear:(BOOL)animated {
       // Interactive transitions can cancel and cause imbalance of will and did calls.
       id<UIViewControllerTransitionCoordinator> coordinator = [self transitionCoordinator];
       if (coordinator && [coordinator initiallyInteractive]) {
-        [coordinator notifyWhenInteractionEndsUsingBlock:
-         ^(id<UIViewControllerTransitionCoordinatorContext> context) {
-           if ([context isCancelled]) {
-             id object =
-                 objc_getAssociatedObject(self, @selector(greyswizzled_viewWillAppear:));
-             UNTRACK_STATE_FOR_OBJECT(kGREYPendingViewsToAppear, object);
-           }
-         }];
+        id block = ^(id<UIViewControllerTransitionCoordinatorContext> context) {
```

Comment:
can you use a stronger type than id for block?

Reply:
Done.